src/errors/configurationError.ts

```diff
@@ -31,10 +32,14 @@ const HandlersByCode = {
   [ConfigurationErrorCode.UnspecifiedUrl]: (params: {
     method: ResolutionMethod;
   }) => `Unspecified url in Resolution ${params.method} configuration`,
-  [ConfigurationErrorCode.MissingProviderConfigurations]: () =>
-    `Couldn't find any configurations\n\tUse -C to configurate the library`,
-  [ConfigurationErrorCode.DependencyMissing]: (params: {dependecy: string, version: string}) => 
-    `Missing dependency for this functionality. Please install ${params.dependecy} @ ${params.version} via npm or yarn`
+  [ConfigurationErrorCode.MissingProviderConfigurations]: 
+  () => `Couldn't find any configurations\n\tUse -C to configurate the library`,
+  [ConfigurationErrorCode.DependencyMissing]: (params: {
+    dependecy: string, version: string
+  }) => `Missing dependency for this functionality. Please install ${params.dependecy} @ ${params.version} via npm or yarn`,
+  [ConfigurationErrorCode.IncorrectBlockchainProvider]: (params: {
```

Comment:
As a client i would like to see original error in case of network issues, not wrapped one